lib/StaticNarrative/exporter/processor_util.py

```diff
@@ -4,17 +4,19 @@
 import html
 import json
 import os
-from pathlib import Path
 from urllib.parse import quote
 
 from installed_clients.authclient import KBaseAuth as _KBaseAuth
 from installed_clients.WorkspaceClient import Workspace
 
+from StaticNarrative import STATIC_NARRATIVE_BASE_DIR
+
 ICON_DATA = None
 
 
 def _load_icon_data():
-    icon_json = os.path.join("/kb", "module", "data", "icons.json")
+    # this should access the local folder
+    icon_json = os.path.join(STATIC_NARRATIVE_BASE_DIR, "data", "icons.json")
```

Comment:
make these locations relative to the StaticNarrative dir, so tests can be run locally as well as on a docker image.